.github/workflows/validate.yml

```
@@ -0,0 +1,31 @@
+# PR validation checks
+
+name: Validate
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Deephaven
+        run: docker-compose -f ./test/docker-compose.yml -p test-tables up -d
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Upgrade pip #For some reason the install fails without this
+        run: pip install --upgrade pip
+      - name: Install python dependencies
+        run: pip install -r ./validate/requirements.txt
```

```suggestion
      - name: Install python dependencies
        run: |
            pip install --upgrade pip
            pip install -r ./validate/requirements.txt
```
I don't think the comment is needed when taken in context of the following step, so I just combined the two.